experiments/create_simulated_body/sim_body_gui.py

```
@@ -0,0 +1,1038 @@
+            'crater_min_radius': self._crater_min_radius,
+            'crater_max_radius': self._crater_max_radius,
+            'crater_power_law_exponent': self._crater_power_law_exponent,
+            'crater_relief_scale': self._crater_relief_scale,
+            'anti_aliasing': self._anti_aliasing,
+            # We intentionally don't save the show_visual_aids and zoom_sharp parameters
+            # so that the JSON file is more easily used in a simulated model file,
+            # but if the user adds these fields to the JSON file, they will be restored
+            # on load.
+        }
+
+    def _set_parameters_from_dict(self, params: dict[str, Any]):
```

Ruff (0.14.4)

Missing return type annotation for private function `_set_parameters_from_dict`

Add return type annotation: `None`

(ANN202)